sql/plsql/PlSqlParser.g4

```diff
@@ -992,6 +992,21 @@ validation_clauses
         online_or_offline? into_clause?
     ;
 
+compute_clauses
+    : COMPUTE SYSTEM? STATISTICS for_clause?
+    ;
+
+for_clause
+     : FOR ( TABLE for_clause*
+           | ALL (INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? for_clause* | LOCAL? INDEXES)
+           | COLUMNS (SIZE UNSIGNED_INTEGER)? for_column_clauses for_clause*
+           )
+    ;
+
+for_column_clauses
+    : column_name SIZE UNSIGNED_INTEGER for_column_clauses*
+    ;
```

Comment:
What do you think about simplification by the following way?

```suggestion
for_clause
     : FOR ( TABLE for_clause*
           | ALL (INDEXED? COLUMNS (SIZE UNSIGNED_INTEGER)? for_clause* | LOCAL? INDEXES)
           | COLUMNS (SIZE UNSIGNED_INTEGER)? (column_name SIZE UNSIGNED_INTEGER)+ for_clause*
           )
    ;
```

It looks like there is no need for separated `for_column_clauses` rule.

Reply:
That makes sense @KvanTTT. Simplified the changes.